CHANGELOG.md

```diff
@@ -162,6 +162,9 @@ Buffers for state serialization instead of Amino.
 * (server) [\#5709](https://github.com/cosmos/cosmos-sdk/pull/5709) There are two new flags for pruning, `--pruning-keep-every` 
 and `--pruning-snapshot-every` as an alternative to `--pruning`. They allow to fine tune the strategy for pruning the state.
 * (crypto/keys) [\#5739](https://github.com/cosmos/cosmos-sdk/pull/5739) Print an error message if the password input failed.
+* (mint) [\#5411](https://github.com/cosmos/cosmos-sdk/issues/5411) Now it uses a moving average to calculate the blocks generated in a year.
```

Comment:
```suggestion
* (mint) [\#5411](https://github.com/cosmos/cosmos-sdk/issues/5411) Inflation is now computed based on a cumulative moving average instead of the (removed) `BlocksPerYear` parameter.
```